coldfront/core/user/models.py

```diff
@@ -6,6 +6,10 @@
 from rest_framework.authtoken.models import Token
 
 
+PHONE_REGEX = RegexValidator(
```

Comment:
Had you taken a look at the `django-phonenumber-field` module recommended in the answer this regex came from? It looks like it takes care of pretty-printing, international numbers, etc., from a brief glance. Thoughts?

Reply:
(Also, there's some conversation about the lite version, which uses less memory with some trade-offs.)

Reply:
oh alright, I thought of using this but decided against adding another dependency... ill push the changes!